sourcegraph/sourcegraph
# [SG-34727] Accessibility: Search box and homepage misc mobile bugs

### Description
- Search context dropdown: Description of search contexts almost completely cut off on mobile devices. 
![Image](https://user-images.githubusercontent.com/206864/166075973-b066be09-329b-4f74-a473-bc907282a272.png)
### Expected Behaviour
- It should be fixed (suggested fix: add a tooltip for the description)

### Refs
[SourceGraph Issue](https://github.com/sourcegraph/sourcegraph/issues/34727)
[GitStart Issue
](https://app.gitstart.com/clients/sourcegraph/tickets/SG-34727)

### Test plan
1. Open the `developer tool` and reduce the screen size to about `300px` mobile size.
2. Click on the `Search Context Menu` for the dropdown to appear.
3. Hover over the `description` that is truncated, the `tool-tip` which is the fix should appear.



## App preview:

- [Web](https://sg-web-contractors-sg-34727.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-kywwrkspos.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
